### ISSUE TITLE
[CAIA Intake] VA BTSSS Travel Pay: Review on VA.gov Travel Pay Page

### ISSUE THREAD

**Issue by ayush-chak** [department-of-veterans-affairs/va.gov-team#79982]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: VA BTSSS (Travel Pay)
- **OCTO product owner:** Kay Lawyer
- **Product name:** Travel Pay on VA.gov (MVP)
- **Product manager:**  Ayush Chakravarty
- **Slack channel:** https://dsva.slack.com/archives/C05UTPZRZFY
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [X] Something else:  Adding a new authenticated page which shows the status of travel claims

### What's the nature of your initiative and desired outcomes?

Creating a prototype to conduct a look-and-feel study to validate finding from a tree study. If validated it will likely move on to engineering implementation.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [X] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Honestly, all of things. We are still discovering what users want, as well as testing with what is available from the begining stages of the new BTSSS API. We are mostly in the later phase of discovery but still in the early phase of wireframing and research.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [X] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `[79471](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79471) `

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708602401825/9786edc7ee2a5b6c4efd1a895d067a07ae2a7f27?wid=0-1709668560371) `Mural board with userflow and mockups`
- [Supporting artifact](https://codepen.io/team/VA-gov-Applications/project/live/DQmqxR) `CodePen prototype`

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
```

```[tasklist]
### CAIA a11y tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/81037
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/80740
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/93915
```

```[tasklist]
### CAIA IA tasks
```

**Comment by carafrissell1** [comment]
Hello! I checking - hoping - to see if we can get Accessibility support for the screen reader users we have participating in the study.  To date we have 3 screen readers scheduled (below) with a 4th yet to be scheduled. 

Friday, April 19, 2024 at 11am EST - 12
Friday, April 26, 2024 at 12:30pm EST - 1:30
Wednesday, May 1 at 11am EST - 12

@coforma-terry

**Comment by laurwill** [comment]
Hey @carafrissell1 @ayush-chak , 
Wanted to check in on this travel pay work — looks like you all did some research that wrapped in early May. What are your next steps? I think this may involve a way to check status of travel pay claims, and we want to look at how those statuses align with statuses on other types of VA claims and forms. Thanks!

**Comment by micahtaylor** [comment]
Hey @laurwill 
The experience is making it's way into dev and can be seen [here](https://dev.va.gov/my-health/travel-claim-status) after logging in with any test user (let me know if you need one).

A couple of items that could use your expertise...

The help content was vetted as part of the governance review but would love if you have any tweaks.

> If you need to manage a claim, log into the [BTSSS portal](https://dvagov-btsss.dynamics365portals.us/)
 or call [855-574-7292](tel:+18555747292) from 7 a.m. to 7 p.m. Monday through Friday. Have your claim number ready to share when you call.
 
<img width="1034" alt="Screenshot 2024-06-06 at 4 51 45 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/671894/cc1d057e-46b7-43b7-a17b-6cf3dde4d9de">

-------

Additionally, I'd appreciate your take on the upcoming feature of adding filters to the page. 
The current filter design can be seen in [this Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708602401825/9786edc7ee2a5b6c4efd1a895d067a07ae2a7f27?wid=0-1717687645892).
User testing showed that users really only wanted to know about claims that they wanted to take action on (`Denied, On hold, In manual review`). I'd like to allow those as filters but "Others" seems like it might be odd even with the help text.

**Comment by laurwill** [comment]
Hey Micah! Looks like you're farther along in the designs that I realized, and I don't think anyone from CAIA has been involved with this new page yet (unless I missed that!). I have a bunch of questions about the statuses and how this page relates to BTSSS — let's find time for a sync to get me up to speed. I'll look for a time tomorrow or next week.

**Comment by micahtaylor** [comment]
Documenting our Slack discussion here as well. The status page on dev is an MVP integration with the BTSSS API. We are in the beginning stages of discovery and research around status language. @carafrissell1 will start the connection with @laurwill over status language. I'll also lead the connection over filtering.

**Comment by carafrissell1** [comment]
Tuesday 6/11 Content / IA Syncin meeting: [@Laura Willwerth](https://dsva.slack.com/team/U03CHM52JBF) , [@Sara Sterkenburg](https://dsva.slack.com/team/U045GGS6V25) [@Ayush Chakravarty](https://dsva.slack.com/team/U06QY8JM29Y)  and Cara Frissell

Notes/collab [in Canvas](https://dsva.slack.com/docs/T03FECE8V/F077MRSM1C2) 


* some design things - the list should follow the box, see Sara’s comments on Slack
* Add a link to place where statuses are defined so that users see that information (to explainer text) (Q: where are those in [va.gov](http://va.gov/)) (see comment below)
* Clarify what can and and cannot be done (in the explainer text) --- [@Laura Willwerth](https://dsva.slack.com/team/U03CHM52JBF)  can do a pass on that 
    * Further context post-conversation: so far have not found a page on [VA.gov](http://va.gov/) that has the definition of each status.  This means we would need to direct to BTSSS.  Takeaway:  roll this into the explainer text? 
* Can we further distinguish between the status and submission --- "Check travel claim reimbursement status" is what was tested 
* Clarifying the links from the MHV page - action: connect eng teams 
    * Should we have the MHV branding? 
    * Connect Kevin with Cartography eng to discuss secondary nav

**Comment by carafrissell1** [comment]
@sterkenburgsara feedback on 6/11 below.  Also see in Slack[ here](https://dsva.slack.com/archives/C05UTPZRZFY/p1718118055893989).  

[@Kevin Duensing](https://dsva.slack.com/team/U01NU8D1Q5B)
 things I'm seeing in staging that maybe you could look into?
Missing pagination. Most teams load 10 items per page, and I'd suggest passing a parameter to load pages in the URL?
There are duplicate "Feedback" buttons
Need to add the [back to top](https://design.va.gov/components/back-to-top) component
H1 (page title) in staging is spanning all 12 grids, but we typically wrap content at 8/12 grids (75%) of the screen. Reason is scannability - it reduces cognitive load when we minimize characters per line.
Update the page title tag to read "Check travel claim status | Veterans Affairs"
Sort label that says "showing 1-150 of 150" needs to come after the sort button and before the list. Right now it is above the sort button / selection dropdown.

**Comment by micahtaylor** [comment]
[Content discussion is available in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708602401825/9786edc7ee2a5b6c4efd1a895d067a07ae2a7f27?sender=u276b0ff8032c789bb6cd2724)

**Comment by laurwill** [comment]
@carafrissell1 is there any work remaining in this intake, or should we close and open new intakes for future stages of travel pay work?
